src/main/scala/dal/table/blockDetailTable.scala

```diff
@@ -0,0 +1,17 @@
+package dal.table
```

Comment:
Very minor point - the actual file names follow the class names so should begin with a capital, e.g. BlockDetailTable.scala.

Nothing bad happens if we leave it as-is though.

Reply:
I've tried, both Sublime and File Explorer automatically change it back to lower case at the start :(

Reply:
Rename it to something else and then back in the case you want